core/primitives/chain/chain.cpp

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "primitives/chain/chain.hpp"
+
+namespace primitives::chain {
+
```

Comment:
is it supposed to be empty?

Reply:
yes, for time being

Reply:
It's only interface for chain manager (fil-85), currently only chain store is implemented